plugins/kubernetes/app/decorators/stage_decorator.rb

```diff
@@ -6,8 +6,8 @@
   after_create :seed_kubernetes_roles
   validate :validate_not_using_non_kubernetes_rollback, if: :kubernetes?
 
-  has_many :kubernetes_roles, class_name: "Kubernetes::StageRole", dependent: :destroy
-  accepts_nested_attributes_for :kubernetes_roles,
+  has_many :kubernetes_stage_roles, class_name: "Kubernetes::StageRole", dependent: :destroy
```

Comment:
inverse_of ?

Reply:
got a test for that :)
when it's obvious rails can figure it out on it's own